config/common/nextflow-aurora-mysql.yaml

```diff
@@ -0,0 +1,7 @@
+template_path: nextflow-aurora-mysql.yaml
+stack_name: nextflow-aurora-mysql
+parameters:
+  VpcID: !stack_output_external nextflow-vpc::VPCId
```

Comment:
why is this in `config/common`?  don't you want to create the DB in both dev and prod accounts?  

This depends on the VPC in nextflow-vpc.yaml file so please add a `dependencies` section for it.   Also `stack_tags` section would be good as well.

Reply:
@zaro0508 You're right, although this works nicely in development due to the way the CICD deploys I'll have to duplicate this. Bummer.

Will add dependencies section. Please indicate which stack_tags you thing are appropriate. This will be shared by many projects.